### PR TITLE
fix: can now delete all payouts

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

_Provide a description of what this PR does. Link to any relevant GitHub issues, Notion tasks or Discord discussions._

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] (if relevant) I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] (if relevant) I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3603** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3603" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3603?utm_source=stack-comment).

File: yarn.lock [hunk 1]
```diff
@@ -2961,9 +2961,9 @@
     "@openzeppelin/contracts" "^4.8.0"
     "@paulrberg/contracts" "^3.7.0"
 
-"@jbx-protocol/juice-ownable@git+https://github.com/jbx-protocol/juice-ownable.git":
```
Comment (wraeth-eth) [inline yarn.lock:2964]: im unsure why I keep getting this file

File: src/components/v2v3/shared/Allocation/AllocationList.tsx [hunk 2]
```diff
@@ -82,7 +83,9 @@ export const AllocationList = ({
       const totalAmount = parseFloat(fromWad(totalAllocationAmount))
       const removedAmount = (allocation.percent / 100) * totalAmount
 
-      const totalAmountAfterRemoval = totalAmount - removedAmount
+      const totalAmountAfterRemoval = roundIfCloseToNextInteger(
```
Comment (wraeth-eth) [inline src/components/v2v3/shared/Allocation/AllocationList.tsx:86]: This was a simple enough fix - basically just sometimes left over with a really really small decimal value that causes an underflow

File: yarn.lock (hunk 1 above)
Comment (johnnyd-eth) [reply yarn.lock:2964]: Can we just remove it from the commit do you think? And have you `yarn pristine`d?

**Review by johnnyd-eth — APPROVED** [review]
It fixes Jango's case so I'll get it in, but this needs further work asap. 

e.g. Right now you still have to do Jango's little trick to bring Peel's distributionLimit to 0, because there is some pay allocated to the project owner (since the recent `editPayouts` where I reduced my pay)

**Pull request merged**